test/tool/test_data/concat_data.json

```diff
@@ -0,0 +1,370 @@
+{
+  "inputsData": {
```

Comment:
These generated test data of `inputsData` and `expectedData` are used as the [data](https://github.com/BruceDai/wpt/blob/add_webnn_concat_tests/webnn/resources/test_data/concat.json#L295) of test_data file for WPT WebNN concat tests.